scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -230,8 +229,17 @@
                     :pod-namespace pod-namespace
                     :task-metadata task-metadata})))))
 
+(defn get-pods-in-pool
+  "Given a compute cluster and a pool name, extract the pods that are in the current pool only."
+  [{:keys [pool->node-name->V1Node node-name->pod-name->V1Pod]} pool]
```

Comment:
I would favor just using `node` and `pod` instead of `V1Node` and `V1Pod`

Reply:
Renamed.